templates/publisher/listing.html

```diff
@@ -318,6 +318,36 @@
             </div>
           </div>
         </div>
+
+        <div class="p-strip is-shallow">
+          <div class="row">
+            <hr class="u-no-margin--bottom" />
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+            <label>License</label>
+          </div>
+          <div class="col-8">
+            <input type="hidden" name="license" value="{{ license }}" />
+            <input type="radio" name="license-type" value="simple" id="license-simple-option" {% if license_type == 'simple' %}checked="checked"{% endif %} />
+            <label for="license-simple-option">Simple</label>
+            <div class="js-license p-multiselect__wrapper" {% if license_type != 'simple' %}style="display: none;"{% endif %} id="license-simple-input">
+              <input type="text" class="js-multiselect-input" name="license-simple" value="{% if license_type == 'simple' %}{{ license }}{% endif %}" />
+              <div class="js-multiselect-holder"></div>
+              <p class="p-form-help-text">The license(s) under which you will release your snap.<br />Multiple licenses can be selected to indicate alternative choices.</p>
+            </div>
+            <input type="radio" name="license-type" value="custom" id="license-custom-option" {% if license_type == 'custom' %}checked="checked"{% endif %} />
+            <label for="license-custom-option">Custom SPDX expression</label>
+            <textarea
```

Comment:
Does it need to be a text area? Wouldn't a text input be enough?

<img width="1038" alt="screen shot 2018-10-02 at 13 05 29" src="https://user-images.githubusercontent.com/83575/46345254-2f64a180-c644-11e8-8efd-857ac4c1a624.png">
Also it could be useful to add some hint text linking to what SPDX expressions are?